Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -142,6 +142,14 @@ extension Event.HumanReadableOutputRecorder {
   }
 }
 
+extension Test {
```

Comment:
I'd prefer to make this just a function local to the file rather than extension because the extension implies something a bit stronger than just "this is the string this recorder uses" to me.

Reply:
Also "title" seems like the wrong noun, but that's probably a bit _too_ nitpicky.

Reply:
Open to suggestions but this is just private to the file so I'm not too concerned by the word "title"

Reply:
I got nothin'.